superset-frontend/plugins/legacy-plugin-chart-partition/src/Partition.js

```diff
@@ -274,7 +273,7 @@ function Icicle(element, props) {
             '<tr>' +
             '<td>' +
             '<div ' +
-            `style='border: 2px solid ${atNode ? 'black' : 'transparent'};` +
+            `style='border: 2px solid ${atNode ? 'white' : 'transparent'};` +
```

Comment:
@michael-s-molina should we use the available white in the theme vars?

Reply:
Nice catch! I removed the condition and used the transparent border to always show the partition's colors. Previously, when the user was hovering over a partition, the leaf was white. Now the leaf obeys the color of the partition. I think it's more intuitive.